examples/flutter_gallery/lib/demo/fortnightly/fortnightly.dart

```diff
@@ -0,0 +1,232 @@
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class FortnightlyDemo extends StatelessWidget {
+  static const String routeName = '/fortnightly';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Typography Demo',
+      theme: _fortnightlyTheme,
+      home: Scaffold(
+        body: Stack(
+          children: <Widget>[
+            FruitPage(),
+            Padding(
+              padding: EdgeInsets.only(top: MediaQuery.of(context).padding.top),
+              child: ShortAppBar(
+                onBackPressed: () {
+                  Navigator.pop(context);
+                },
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class ShortAppBar extends StatelessWidget {
+  const ShortAppBar({this.onBackPressed});
+
+  final VoidCallback onBackPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 96,
+      height: 50,
+      child: Material(
+        elevation: 4,
+        shape: const BeveledRectangleBorder(
+          borderRadius: BorderRadius.only(bottomRight: Radius.circular(22)),
+        ),
+        child: Row(
+          children: <Widget>[
+            IconButton(
+              icon: const Icon(Icons.arrow_back),
+              onPressed: onBackPressed,
+            ),
+            const SizedBox(width: 12),
+            Image.asset(
+              'logos/fortnightly/fortnightly_logo.png',
+              package: 'flutter_gallery_assets',
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class FruitPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final TextTheme textTheme = Theme.of(context).primaryTextTheme;
+
+    return SingleChildScrollView(
+      padding: EdgeInsets.only(bottom: MediaQuery.of(context).padding.bottom),
+      child: Container(
+        color: Colors.white,
+        child: Column(
+          children: <Widget>[
+            Container(
+              constraints: const BoxConstraints.expand(height: 248),
+              child: Image.asset(
+                'food/fruits.png',
+                package: 'flutter_gallery_assets',
+                fit: BoxFit.fitWidth,
+              ),
+            ),
+            const SizedBox(height: 17),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 8),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Text(
+                        'US',
+                        style: textTheme.overline,
+                      ),
+                      Text(
+                        ' ¬ ',
+                        style: textTheme.overline.apply(
```

Comment:
Not sure why we're looking up the primary color this way? Couldn't this be a ThemeData.colorScheme lookup?

Also: the method already binds `textTheme` and we want the closing parens to line up:

```
style: textTheme.overline.apply(
  textTheme.display3.color,
)
```
This is OK too: 
```
style: textTheme.overline.apply(color: textTheme.display3.color),
```

Reply:
We could use a ColorScheme but I don't have that information from a designer. I could make a TODO for it tho.